test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- [ ] It is nice , you are create this file but please this file should be ignore in your git ignore file.

Reply:
I had some problems fixing this point because .gitignore was ignoring the "test.md" file. That's why I had to change the name to "testing.md" but finally is working now.